src/lib/api/postsFetcher.ts

```diff
@@ -1,10 +1,11 @@
 export async function fetchPosts(
   category: string | null = "",
-  tag: string | null = ""
+  tag: string | null = "",
+  author: string | null = ""
```

Comment:
Are these params expected to be null?

Reply:
No but what if they are null in the database? How can we be sure that the entries in database are non-null? So just to be sure I've provided a null type. If there's any other reason I'll remove them.